firmware/octopi_firmware_v2/main_controller_teensy41/main_controller_teensy41.ino

```diff
@@ -12,7 +12,10 @@
 //#include "def_platereader.h"
 //#include "def_squid_vertical.h"
 
-#define N_MOTOR 3
+// the firmware support axises number
+#define N_MOTOR 4
+// default axises, such as X,Y,Z
+#define STAGE_AXIS 3
```

Comment:
Same as the above reply.

Reply:
This was a separate grammar request.  "Axis" is a singular noun.   The plural form is "Axes".  So this should be `STAGE_AXES`

Reply:
@veerwang I don't see an "reply above".

Spelling: axises -> axes.

Why adding a new constant `STAGE_AXIS`?

Reply:
I would make axises -> axes

Reply:
In the default situation there are only three motor drivers board in the device. But there are four slots for motor drivers boards in the device. For compatible reason, for default initializing only three drivers variables have been processing, so I used the STAGE_AXES constant to control it. But in the firmware it would keep four variables for recording the axes status, once users initialize the forth one, so I used the N_MOTOR to control it.